crates/ruff_python_parser/src/lexer.rs

```diff
@@ -1247,14 +1247,29 @@ mod tests {
     const MAC_EOL: &str = "\r";
     const UNIX_EOL: &str = "\n";
 
-    fn lex_source(source: &str) -> Vec<Tok> {
-        let lexer = lex(source, Mode::Module);
-        lexer.map(|result| result.unwrap().0).collect()
+    #[derive(Debug, PartialEq)]
+    struct Token {
+        value: Tok,
+        range: TextRange,
     }
```

Comment:
What's the reason for using `Token` vs `Spanned` directly?

Reply:
It must've been an oversight. I think `Spanned` is a more concise representation as you've also pointed out in your other comment :)